Source/TaggingCriteria/ChiExtractionTaggingCriterion.hpp

```diff
@@ -42,7 +42,7 @@ class ChiExtractionTaggingCriterion
     // The constructor
     ChiExtractionTaggingCriterion(const double dx, const int a_level,
                                   const int a_max_level,
```

Comment:
I think `max_level` is not being used in this criterion

Reply:
Yes, I think this was accidentally added in #75. I won't change it to maintain backwards compatibility.